src/sidebar/services/test/session-test.js

```diff
@@ -172,30 +172,26 @@ describe('SessionService', () => {
 
       it('should cache the returned profile data', () => {
         const session = createService();
-        return session
-          .load()
-          .then(() => {
-            return session.load();
-          })
-          .then(() => {
-            assert.calledOnce(fakeApi.profile.read);
-          });
+        session.load();
```

Comment:
This change is unrelated, but I think it is simpler. Two consecutive calls to `session.load` doesn't invoke the reading of the profile because it is cached for X amount of time.